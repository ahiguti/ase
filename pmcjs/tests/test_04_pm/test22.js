
use('Perl');
Perl.Use('Digest');
var funcs = ['MD5', 'SHA1', 'HMAC_MD5', 'HMAC_SHA1'];
var str = "foobar";
for (var i in funcs) {
  var f = funcs[i];
  var ctx = new Perl('Digest', f);
  ctx.add(str);
  var r = ctx.hexdigest();
  print(r);
}
