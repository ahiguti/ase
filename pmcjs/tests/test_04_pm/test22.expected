3858f62230ac3c915f300c664312c63f
8843d7f92416211de9ebb963ff4ce28125932878
425980b11f70991f4249a68c754357d4
168197ca3a9d4fe60ede05116b0a125da1ab3bd0
